deploy_snapshot_and_javadoc.sh

```diff
@@ -24,4 +25,26 @@ else
   echo "Deploying snapshot..."
   ./gradlew clean uploadArchives
   echo "Snapshot deployed!"
+
+  echo "Uploading javadoc..."
+
+  ./gradlew aggregatedJavadocs
+
+  {
+    set +x
+    cd $HOME
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "travis-ci"
+    git clone --quiet https://${GH_TOKEN}@github.com/wala/javadoc > /dev/null
+    set -x
+  }
+  cd javadoc
+  git rm -rf --quiet *
```

Comment:
Prefer `./*` instead of `*` so that filenames starting with `-` will not be treated as options. We currently have no files with names like that, but it is wise to be paranoid when recursively, forceably removing files.